database_scripts/tabels/create_product.sql

```diff
@@ -1,14 +1,14 @@
 use customer_db
 
-create table dbo.Product
-(
-ID int PRIMARY KEY FOREIGN KEY dbo.Product,
-CategoryId int FOREIGN KEY dbo.Category,
-Name nvarchar(50),
-Description nvarchar(max),
-Price decimal(10,2),
-QuantityInStock int,
-Availablity bit,
-UpdatedDate datetime,
-CreatedDate datetime
+CREATE TABLE [dbo].[Products] (
+    [ID] [int] NOT NULL IDENTITY,
+    [Category] [int],
+    [Name] [nvarchar](max),
+    [Description] [nvarchar](max),
+    [Price] [decimal](18, 2),
+    [Quantityinstock] [int],
+    [Avilability] [bit],
+    [UpdatedDate] [datetime],
+    [CreatedDate] [datetime],
+    CONSTRAINT [PK_dbo.Products] PRIMARY KEY ([ID])
 )
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired(), długości nvarcharów np. [FirstName] [nvarchar](100), zmień `use customer_db` na `use BProject_db`